nwb.config.js

```diff
@@ -0,0 +1,23 @@
+const browsers = require('./test/browsers');
```

Comment:
I haven't seen that one. Looks promising. If it makes sense, yeah, worth a change. Investing in a tool that you have minimal configuration for means that switching is cheap, especially if you haven't exposed your directory structure as API (using "main" and "modules").